package.json

```diff
@@ -148,6 +148,7 @@
 	},
 	"devDependencies": {
 		"@automattic/calypso-build": "1.0.0",
+		"@automattic/calypso-color-schemes": "1.0.0",
```

Comment:
Isn't this now `dependencies` since it's used in the code, not just `devDependencies`?

Reply:
It's used at the same time as `calypso-build` 🤷‍♂ 

I think the distinction and separation is nonsensical in this package.json.

What's the production app? Jetpack the plugin? That doesn't rely on any dependencies (node_modules) at run time.

Or is the app this package.json describes a Jetpack builder? In that case everything is a dependency and what are devDependencies for? (I lean towards this thinking).

I think the distinction is mostly useful for distributed modules, which this is not.

For cleanliness and consistency, I'd put all of the dependencies in this package.json into `dependencies` and call it a day. But I don't want to do that in this PR and I don't think it makes any difference where the dependency is for this app.

Reply:
>Or is the app this package.json describes a Jetpack builder? In that case everything is a dependency and what are devDependencies for? (I lean towards this thinking).

Oh yeah I'm in this camp totally too. :-( I use the distinction myself just to communicate how something is used in the repo and practically everything always gets installed anyway.

No strong opinions where it is here since it doesn't indeed have difference in practise. :-)